src/main/java/teammates/ui/controller/StudentCourseJoinAuthenticatedAction.java

```diff
@@ -71,10 +71,10 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         response.addResponseParam(Const.ParamsNames.CHECK_PERSISTENCE_COURSE, getStudent().course);
         excludeStudentDetailsFromResponseParams();
 
-        if (statusToAdmin == null || statusToAdmin.trim().isEmpty()) {
-            statusToAdmin = studentInfo;
+        if (statusToAdmin == null || statusToAdmin.isEmpty()) {
+            statusToAdmin.add(studentInfo);
         } else {
-            statusToAdmin += "<br><br>" + studentInfo;
+            statusToAdmin.add("<br><br>" + studentInfo);
```

Comment:
The whole if statement can be removed and replaced with 
```java
statusToAdmin.add(studentInfo);
```
since statusToAdmin is a list, and is initialized with a default value.